src/FooterList/index.tsx

```diff
@@ -0,0 +1,39 @@
+import { breakpoint, color, fontWeight, space } from '@core-ds/primitives';
+import * as React from 'react';
+import styled from 'styled-components';
+import { Link } from '../Link';
+
+export type FooterListItemProps = React.PropsWithChildren<{
+   className?: string;
+}>;
+
+export const FooterListItem = styled.li<FooterListItemProps>`
+   display: flex;
+   align-items: center;
+   padding-top: ${space[1]};
+   padding-bottom: ${space[1]};
+   & > :not(:first-child) {
+      margin-left: ${space[2]};
+   }
```

Comment:
This rule applies to the children of the list item. It was meant to add space between an icon and the link label (for social links). I've since changed the link implementation so I'm going to clean up this as it's no longer necessary